src/wolfio.c

```diff
@@ -151,7 +151,21 @@ static int TranslateIoError(int err)
         return WOLFSSL_CBIO_ERR_CONN_CLOSE;
     }
 
+#if defined(_WIN32) && defined(DEBUG_WOLFSSL)
+	char errstr[1024] = { "\tGeneral error: "};
```

Comment:
Section should be indented with 4 spaces. This section appears to be one too far to the right.